webpack/webpack.common.js

```diff
@@ -49,4 +49,11 @@ module.exports = {
       ],
     }),
   ],
+  devServer: {
```

Comment:
@devpurohit Can this be added to the [webpack.dev.js](https://github.com/sydrawat/react-boilerplate/blob/master/webpack/webpack.dev.js) file at [line 33](https://github.com/sydrawat/react-boilerplate/blob/master/webpack/webpack.dev.js#L33)?